src/api/hooks/storageProviders.ts

```diff
@@ -15,7 +15,7 @@ export const useStorageProvider = (id: string, opts?: StorageProviderOpts) => {
     variables: { where: { id } },
   })
   return {
-    storageProvider: data?.storageProviderByUniqueInput,
+    storageProvider: data?.workerByUniqueInput,
```

Comment:
I'm not entirely sure about this. It can be confusing in the future. Maybe, we should also change the name of`storageProviders`(and corresponding hooks) to just `workers`?

Reply:
It makes sense this way, because storage providers are only one type of worker. And here we are interested only in storage providers. Will add additional query filter by type